### ISSUE TITLE
[Bug] Ctl A twice + Ctl-V doesn’t replace the whole content of the page

### ISSUE THREAD

**Issue by svengau** [codex-team/editor.js#1705]
Copy some text, then on official demo, select all content (ctrl A + ctrl A) and try to paste. Nothing happens.

This is a minor bug, because a quick workaround is to delete the text prior to paste the content of the clipboard.

Steps to reproduce:

1. Copy some text
2. Go to https://editorjs.io
3. Select all content, by typing ctrl A + ctrl A
4. Try to paste

Device, Browser, OS: Macbook Pro, Chrome 91, Catalina

Editor.js version: the one on the official website

**Comment by maziyank** [comment]
Me also experience this. Happened when I try to copy list block and then Ctrl+A and Ctrl+V to other block.

**Comment by Y0ursTruly** [comment]
I don't think this problem doesn't exist anymore.. just checked.. also happy new year

**Comment by svengau** [comment]
Sorry, but I can still reproduce this issue. Select all content, and try to paste, it doesn't work. (but it work if you select just a block).

ps: Happy new year too.
